README.md

```diff
@@ -102,10 +102,17 @@ function mapActionToProps(dispatch) {
 				data: { todo }
 			})
 		}
-	}
+	};
+}
+
+function mergeProps(stateProps, actionsProps) {
```

Comment:
can you write another block of documentation for mergeProps rather than putting it in the here? some users will think its a required parameter :)